.github/workflows/delete_images.yaml

```diff
@@ -0,0 +1,24 @@
+name: Delete old container images
+
+on:
+  schedule:
+    - cron: '0 0 * * 2' # This job runs every Tuesday
+  workflow_dispatch:
+  push:
+    branches:
+      - ghcr-clean
```

Comment:
Unsure about why there is a `push` trigger on the `ghcr-clean` branch: as noted previously, I don't think we want to trigger the action on push to any branch, but just run on a schedule and/or manually.

Reply:
Hi @esune i am testing stuff so just testing on my branch